content/reference/runtime/nouns.md

```diff
@@ -9,30 +9,35 @@ with nouns is in `u3`.
 ### u3: a map of the system
 
 There are two kinds of symbols in `u3`: regular and irregular.
+All files listed below are found in the `pkg/noun` directory.
```

Comment:
Can we add a link to that directory on GitHub i.e. https://github.com/urbit/vere/tree/develop/pkg/noun?

Reply:
Awesome, thanks @mcevoypeter. This PR is ready for review now. See the vercel bot's link below to see a deployed preview.